docs/nitpick-exceptions

```diff
@@ -16,3 +16,5 @@ py:class a set-like object providing a view on D's keys
 py:class an object providing a view on D's values
 py:class None.  Remove all items from D.
 py:class a shallow copy of D
+py:class v, remove specified key and return the corresponding value.
+py:class None.  Update D from dict/iterable E and F.
```

Comment:
```suggestion
py:class Update D from mapping/iterable E and F
```
From Shane

Reply:
Thanks @nabobalis and @samaloney Doesn't seem to have fixed it though...  Any other ideas?